CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 2022-04-12 HFI Calc - Refactor Request & Response Structure [#1555](https://github.com/bcgov/wps/issues/1555)
```

Comment:
I mean - we don't - but if we're going to use a system that pulls from the last commit message, it would probably end up in there? I don't have answers.